static/app/utils/analytics/trackAdvancedAnalyticsEvent.tsx

```diff
@@ -8,33 +8,37 @@ import {
   performanceEventMap,
   PerformanceEventParameters,
 } from './performanceAnalyticsEvents';
+import {profilingEventMap} from './profilingAnalyticsEvents';
 import {samplingEventMap, SamplingEventParameters} from './samplingAnalyticsEvents';
 import {searchEventMap, SearchEventParameters} from './searchAnalyticsEvents';
 import {settingsEventMap, SettingsEventParameters} from './settingsAnalyticsEvents';
 import {TeamInsightsEventParameters, workflowEventMap} from './workflowAnalyticsEvents';
 
-type EventParameters = GrowthEventParameters &
-  IssueEventParameters &
-  PerformanceEventParameters &
-  DashboardsEventParameters &
-  DiscoverEventParameters &
-  TeamInsightsEventParameters &
-  SearchEventParameters &
-  SettingsEventParameters &
-  CoreUIEventParameters &
-  SamplingEventParameters;
+interface EventParameters
```

Comment:
@scefali I changed this so that in case we ever have name collisions it will properly error. Before this change, it would just merge the two types (and possibly result in a `never` type)